Gemfile

```diff
@@ -265,6 +270,14 @@ group :test do
   gem "database_cleaner" ,  "1.4.1"
   gem "selenium-webdriver", "2.45.0"
 
+  gem "cucumber-api-steps", "0.13", require: false
+  gem "json_spec", "1.1.4"
+
+  source "https://rails-assets.org" do
+    gem "rails-assets-jquery-simulate",     "1.0.1"
+    gem "rails-assets-jquery-simulate-ext", "1.3.0"
+  end
```

Comment:
Possible reintroduced during the rebase.

    source "https://rails-assets.org" do
        gem "rails-assets-jquery-simulate",     "1.0.1"
        gem "rails-assets-jquery-simulate-ext", "1.3.0"
    end

Shouldn't be here, that's it ?